src/modules/logger/logger.cpp

```diff
@@ -639,6 +639,7 @@ void Logger::add_vision_and_avoidance_topics()
 {
 	add_topic("collision_constraints");
 	add_topic("obstacle_distance_fused");
+	add_topic("onboard_computer_status", 200);
```

Comment:
I don't think we need to log it at this rate. 10hz should be more than enough.

Reply:
@TSC21 Isn't the 200ms (5Hz)  more desirable than 10 Hz?

Reply:
Yeah never mind. The `orb_set_interval()` expects an interval in msecs, not a frequency. So yes this is fine.